package.json

```diff
@@ -40,31 +39,27 @@
     "jest": "yarn test:jest",
     "test": "yarn test:jest",
     "test:jest": "env BABEL_ENV=transpilation jest --verbose --coverage",
-    "transpile": "env BABEL_ENV=transpilation babel react/ --out-dir transpiled/react --copy-files --ignore '**/test/**,**/__mocks__/**,**/*.spec.*'",
+    "transpile": "env BABEL_ENV=transpilation babel react/ --out-dir transpiled/react --copy-files --ignore '**/test/**,**/__mocks__/**,**/*.spec.*' --verbose",
```

Comment:
Ca permet de voir quoi en plus le --verbose ?

Reply:
The last version of babel doesn't show transpiled components last before, we have to add the `--verbose` option